agents-api/agents_api/web.py

```diff
@@ -98,7 +99,10 @@ async def _handler(request: Request, exc: Exception):
                 offending_input = offending_input[loc]
 
                 # Keep only the message from the error
-                errors = [error.get("msg", error) for error in errors]
+                errors = [
+                    error.get("msg", error) if hasattr(error, "msg") else error
```

Comment:
This should be `error.get(“msg”, error) if isinstance(error, dict) else getattr(error, “msg”, error)`

Reply:
@Ahmad-mtos